src/scripts/main.js

```diff
@@ -0,0 +1,104 @@
+import 'regenerator-runtime/runtime';
+import {
+    getCartCount, setCartCount, renderCartQuantity, setCartItems,
+    addItemCartSession, openCart, closeCart
+} from './cart';
+
+async function getCategories() {
+    let data = sessionStorage.getItem('categories');
+    if (!data) {
+        data = await fetch('http://localhost:5000/categories');
+        data = await data.json();
+        data.sort((a, b) => a.order - b.order);
+        sessionStorage.setItem('categories', JSON.stringify(data));
+    }
+    else {
+        data = JSON.parse(data);
+    }
+    const mainDiv = document.querySelector('.categories-row');
+    let odd = true;
+    for (let element of data) {
+        if (element.enabled) {
+            const divElement = document.createElement('div');
+            divElement.classList.add('col-12', 'category-item');
+            const imgDiv = document.createElement('div');
+            const image = document.createElement('img');
+            image.src = element.imageUrl;
+            image.height = '200';
+            image.alt = element.key;
+            imgDiv.append(image);
+            const textDiv = document.createElement('div');
+            textDiv.classList.add('flexbox-vertical-horizontal-center', 'px-3');
+            const h3 = document.createElement('h3');
+            h3.textContent = element.name;
+            const p = document.createElement('p');
+            p.textContent = element.description;
+            const button = document.createElement('button');
+            button.classList.add('btn', 'btn-danger', 'explore-button');
+            button.type = 'button';
+            button.id = element.id;
+            button.textContent = `Explore ${element.key}`;
+            textDiv.append(h3, p, button);
+            if (odd) {
+                divElement.append(imgDiv, textDiv);
+            }
+            else {
+                divElement.append(textDiv, imgDiv);
+            }
+            mainDiv.append(divElement);
+            odd = !odd;
+        }
+    }
+    buttonClickInit();
+}
+
+function buttonClickInit() {
```

Comment:
Please name it as per the function it performs, or the behaviour it attaches to the buttons